src/macrofier/engine.mjs

```diff
@@ -627,12 +646,11 @@ function generateSchemas(json, templates, options) {
 
   const schemas = (options.section.includes('methods') ? (hasMethodsSchema(json) ? json.methods : '') : (json.definitions || (json.components && json.components.schemas) || {}))
 
-  const generate = (name, schema, uri, { prefix = '' } = {}) => {
+  const generate = (name, schema, uri, { prefix = '', type = '' } = {}) => {
```

Comment:
this is part of the `polymorphic-reduce` work right? if so, same comment.

Reply:
yes, so moved to separate branch